Formula/eventql.rb

```diff
@@ -0,0 +1,29 @@
+class Eventql < Formula
+  desc "Database for large-scale event analytics"
+  homepage "https://eventql.io"
+  url "https://github.com/eventql/eventql/releases/download/v0.3.2/eventql-0.3.2.tgz"
+  sha256 "d235f3e78fa5f6569fc2db94161d3e3f9cb71dc0646e341acd91814cefd23640"
+  head "https://github.com/eventql/eventql.git"
+
+  depends_on "autoconf" => :build if build.head?
+  depends_on "automake" => :build if build.head?
+  depends_on "libtool" => :build if build.head?
+
+  def install
+    ENV.deparallelize
```

Comment:
Without `ENV.deparallelize` I ran into this issue: https://github.com/eventql/eventql/issues/180